Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide.hs

```diff
@@ -206,9 +207,10 @@ endRide handle@ServiceHandle {..} rideId req = withLogTag ("rideId-" <> rideId.g
                pickupDropOutsideOfThreshold = Just pickupDropOutsideOfThreshold
               }
     -- we need to store fareParams only when they changed
-    endRideTransaction (cast @DP.Person @DP.Driver driverId) booking.id updRide mbUpdatedFareParams booking.riderId
+    (startLocationCustomer, endLocationCustomer) <-
+      endRideTransaction (cast @DP.Person @DP.Driver driverId) booking.id updRide mbUpdatedFareParams booking.riderId pickupDropOutsideOfThreshold tripEndPoint
 
-    notifyCompleteToBAP booking updRide newFareParams
+    notifyCompleteToBAP booking updRide newFareParams startLocationCustomer endLocationCustomer
```

Comment:
why `startLocation` is sent from here?

Reply:
If pickuplocation changes so that is updated in driver side and we send it to rider side for making sure that rideList which gives rides shows actual start and end location

Reply:
That is fine but you may send it at `startRide` itself

Reply:
@umangj-28 @shanm-16 we can send start location in rideStart . Can you please make this change